src/murmur/Meta.cpp

```diff
@@ -747,8 +749,25 @@ void Meta::killAll() {
 	qhServers.clear();
 }
 
+/**
```

Comment:
We use `///` for doxygen comments and we usually keep the documentation of a function at the definition (in the header file) of it and not at its implementation.